channel-api/src/api/use_cases.py

```diff
@@ -0,0 +1,238 @@
+import uuid
```

Comment:
import doesn't seem to be sorted as they usually are (standard library, 3rd party, own imports)

Reply:
1. uuid - standard
1. http - standard
1. urllib - standard
1. web3 - third party
1. box - third-party
1. requests - third-party 
1. libtrustdbirdge - third-party
1. marshmallow - third party